src/app/shared/address-input/address-input.component.ts

```diff
@@ -128,7 +132,8 @@ export class AddressInputComponent implements OnInit, AfterViewInit, OnDestroy,
     private localStorageService: LocalStorageService,
     public langService: LanguageService,
     private store: Store,
-    private cdr: ChangeDetectorRef
+    private cdr: ChangeDetectorRef,
```

Comment:
[Member 'cdr: ChangeDetectorRef' is never reassigned; mark it as `readonly`.](https://sonarcloud.io/project/issues?pullRequest=3427&open=AZLDqqqm8Z0b88jeJjhf&id=ita-social-projects-green-city-client)
Also pay attention on sonar issues